tests/unit/Type/ClassStringTypeTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Tests\Unit\Type;
+
+use Psl\Collection;
+use Psl\Type;
+
+final class ClassStringTypeTest extends TypeTest
+{
+    public function getType(): Type\TypeInterface
+    {
+        return Type\class_string(Collection\CollectionInterface::class);
+    }
+
+    public function getValidCoercions(): iterable
+    {
+        yield [$_ = Collection\Vector::class, $_];
+        yield [$_ = Collection\MutableVector::class, $_];
+        yield [$_ = Collection\Map::class, $_];
+        yield [$_ = Collection\MutableMap::class, $_];
+    }
+
+    public function getInvalidCoercions(): iterable
+    {
+        yield [null];
+        yield [STDIN];
+        yield ['hello'];
```

Comment:
Maybe instead of 'hello' -> Could you set the value to something like : 'UnknownClass' ?
Cause that makes it clear why this fails.

Reply:
Indeed, done.